include/c#/10/HsBuildCodes.csproj

```diff
@@ -19,11 +20,16 @@ Replaces Hardstuck.GuildWars2.Builds.
 		<Copyright>Apache-2.0</Copyright>
 		<PackageLicenseFile>LICENSE</PackageLicenseFile>
 		<RepositoryType>git</RepositoryType>
-		<RepositoryUrl>https://github.com/HardstuckGuild/HsBuildCodes</RepositoryUrl>
+		<RepositoryUrl>https://github.com/HardstuckGuild/HsBuildCodes.git</RepositoryUrl>
+		<PackageProjectUrl>https://github.com/HardstuckGuild/HsBuildCodes</PackageProjectUrl>
+		<EnableNETAnalyzers>true</EnableNETAnalyzers>
```

Comment:
I did not know it was enabled by default, since we are targetting C# 10 I think we can assume it will never target lower C# anyway, so I will remove it.